### ISSUE TITLE
[BUG/Help] <title> 用官方例子训练之后反而不如原先的，哪里错了吗？

### ISSUE THREAD

**Issue by alexhmyang** [THUDM/ChatGLM-6B#483]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

![image](https://user-images.githubusercontent.com/10215059/230804100-c33251d4-ec64-4763-8c99-0f190271a0f0.png)

用官方例子训练之后反而不如原先的，哪里错了吗？

实现不了 官方的结果

### Expected Behavior

https://github.com/THUDM/ChatGLM-6B/blob/main/ptuning/README.md

例子
示例1
Input: 类型#上衣*材质#牛仔布*颜色#白色*风格#简约*图案#刺绣*衣样式#外套*衣款式#破洞
Label: 简约而不简单的牛仔外套,白色的衣身十分百搭。衣身多处有做旧破洞设计,打破单调乏味,增加一丝造型看点。衣身后背处有趣味刺绣装饰,丰富层次感,彰显别样时尚。
Output[微调前]: 这件上衣的材质是牛仔布,颜色是白色,风格是简约,图案是刺绣,衣样式是外套,衣款式是破洞。
Output[微调后]: 这是一款简约的牛仔外套,破洞设计,将牛仔布破洞,带来一种随意与个性。破洞的牛仔外套,展现出时尚气息,带来一种休闲感。同时,刺绣图案,让整件外套更加立体。

### Steps To Reproduce

https://github.com/THUDM/ChatGLM-6B/blob/main/ptuning/README.md

### Environment

```markdown
- OS:
- Python:
- Transformers:
- PyTorch:
- CUDA Support (`python -c "import torch; print(torch.cuda.is_available())"`) :
```

**Comment by xv994** [comment]
训练这事比较玄学，如果确定自己训练步骤没出问题，可以再训练一遍
